cypress/integration/parallel-2/braze.spec.js

```diff
@@ -22,7 +23,7 @@ describe('Braze messaging', function () {
 	});
 
 	const url =
-		'https://m.code.dev-theguardian.com/games/2018/aug/23/nier-automata-yoko-taro-interview';
+		'https://theguardian.com/games/2018/aug/23/nier-automata-yoko-taro-interview';
```

Comment:
whats the reasoning behind this change?

Reply:
Code is not a stable environment for testing against, and my assumption was this was left in from testing a non-live version of Frontend that was required for the test. As the test passes on PROD, I switched it over to the more realistic environment for the user.